src/microbe_stage/MicrobeStage.tscn

```diff
@@ -1042,9 +1061,6 @@ margin_bottom = 218.0
 rect_min_size = Vector2( 252, 0 )
 size_flags_horizontal = 0
 size_flags_vertical = 0
-__meta__ = {
-"_edit_use_anchors_": false
-}
```

Comment:
Is this change intended?

Reply:
I always let these changes through, I don't know what they do but Godot seems to automatically just do these randomly, and it doesn't seem to effect anything.